@shared/api/types.ts

```diff
@@ -21,7 +21,10 @@ export interface Response {
   url: string;
 }
 
-export interface ExternalRequest {
+export interface LyraApiRequest {
```

Comment:
weirdly i hadn't hit this TS error before, but just started getting it - so fixed!